### PR TITLE
Completed the Session and Local storage assignment

### PR DESCRIPTION
**Pull request by shubhambhopale**
Please refer documentation here:
https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage
https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage

### CODE REVIEW

File: session-local-storage.assignment4.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            // isNaN is used to avoid any unprecedented behavior of the counters
+            if (elementId === 'scount') {
+                oldCount = isNaN(parseInt(sessionStorage.getItem('session_count'), 10)) ? 0 : parseInt(sessionStorage.getItem('session_count'), 10);
+            } else {
+                oldCount = isNaN(parseInt(localStorage.getItem('local_count'), 10)) ? 0 : parseInt(localStorage.getItem('local_count'), 10);
+            }
+
+            let newCount = oldCount + 1; // isNaN(sessionStorage.getItem('session_count') ? 0 : sessionStorage.getItem('session_count'))
+
+            elementId === 'scount' ? sessionStorage.setItem('session_count', newCount) : localStorage.getItem('local_count', newCount);
```
Comment (skimran2708) [inline session-local-storage.assignment4.html]: localStorage is not being set

Comment (shubhambhopale) [reply session-local-storage.assignment4.html]: @skimran2708 it's being set. If you have observed I have used ternary operator here. The second part is setting the localStorage.

Comment (skimran2708) [reply session-local-storage.assignment4.html]: @shubhambhopale you are getting the data, not setting the local storage. Look at the second part carefully.

Comment (shubhambhopale) [reply session-local-storage.assignment4.html]: Oh sorry. it Should be 'setItem'. Thanks! I will correct it and check.

**Comment by skimran2708** [comment]
@shubhambhopale Created new PR with requested changes
PR #5

**Pull request closed without merging**